app/waf/src/datasources/bigip.datasource.json

```diff
@@ -25,6 +25,53 @@
           "cred64en"
         ]
       }
+    },
+    {
```

Comment:
It's OK to do it like this. 
Another option is to combine the uploadDO and installDO together, because they are both POST. 
We can organize `headers` within our code(in the uploadDO and installDO functions.)

It's OK for now.

Reply:
Let's keep this at this time and discuss this further later.